roles/apim-analytics-dashboard/templates/carbon-home/conf/dashboard/deployment.yaml.j2

```diff
@@ -392,3 +358,41 @@ wso2.transport.http:
       keyStoreFile: "{{ key_store_location }}"
       keyStorePassword: {{ key_store_password }}
       certPass: {{ cert_password }}
+
+## Dashboard data provider authorization
+data.provider.configs:
+  authorizingClass: org.wso2.carbon.dashboards.core.DashboardDataProviderAuthorizer
+
+## Additional APIs that needs to be added to the server.
+## Should be provided as a key value pairs { API context path: Microservice implementation class }
+## The configured APIs will be available as https://{host}:{port}/analytics-dashboard/{API_context_path}
+additional.apis:
+  /apis/analytics/v1.0/apim: org.wso2.analytics.apim.rest.api.proxy.ApimApi
+
+## Authentication configuration
+auth.configs:
+  type: apim
+  ssoEnabled: true
+  properties:
+    adminScope: apim_analytics:admin_carbon.super
+    allScopes: apim_analytics:admin apim_analytics:product_manager apim_analytics:api_developer apim_analytics:app_developer apim_analytics:devops_engineer apim_analytics:analytics_viewer apim_analytics:everyone openid apim:api_view apim:subscribe
+    adminServiceBaseUrl: {{ admin_service_url }}
+    adminUsername: admin
```

Comment:
@Savidude shall we parameterize the username and password values as well?

Reply:
Fixed https://github.com/wso2/ansible-apim/pull/92/commits/33592d18770430db5df84a8aabbb7aca1c6b2b3c